beeai/Containerfile

```diff
@@ -20,6 +23,10 @@ COPY beeai-gemini-malformed-function-call.patch /tmp
 # https://github.com/i-am-bee/beeai-framework/issues/1064
 COPY beeai-propagate-mcp-errors.patch /tmp
 
+# Add Red Hat internal certificate authority
+COPY certs/Current-IT-Root-CAs.pem /etc/pki/ca-trust/source/anchors/
```

Comment:
adding the internal cert to the container image isn't awesome but short of forwarding `CA_REQUESTS_BUNDLE` `SSL_CERT_FILE` etc i'm not sure of a better approach.